lib/src/components/menu/context_menu.dart

```diff
@@ -452,20 +453,22 @@ class _ContextMenuState extends State<ContextMenu> {
     final bool enableLongPress = platform == TargetPlatform.iOS ||
         platform == TargetPlatform.android ||
         platform == TargetPlatform.fuchsia;
-    return GestureDetector(
-      behavior: widget.behavior,
-      onSecondaryTapDown: (details) {
-        _showContextMenu(
-            context, details.globalPosition, _children, widget.direction);
-      },
-      onLongPressStart: enableLongPress
-          ? (details) {
+    return widget.enabled
+        ? GestureDetector(
```

Comment:
This would cause the widget tree to change and it will rebuild the whole child widget.
I'd suggest to just set the onSecondaryTapDown and onLongPressStart to null if not enabled,
```dart
onSecondaryTapDown: !enabled ? null : (details) { ... }
```

Reply:
* onSecondaryTapDown is null if not enabled
* onLongPressStart is not null if enableLongPress and enabled